### DISCUSSION TITLE
Will there be support for Windows 10 LTSC IoT until 2032?

### DISCUSSION THREAD

**Discussion by Mewto7** [hsoft/dupeguru#993] [Q&A] [answered]
Mainstream support for Windows 10 will end in 2025. After that, the IoT version will be supported until 2032. I realize that is a long time ahead but I just wanted to know if there have been any discussions on whether DupeGuru will continue to support it. I love this software but do not want to move on to Windows 11. 

You people made an awesome software. Thank you.

**Comment by arsenetar** [comment, marked as answer]
Well the only reason to pull support of Windows 10 will be with python versions dropping support for it (ex Python 3.10.x only supports Windows 8+).  New python versions seem to bump Windows version requirements after the Windows version is no longer supported.  The builds for dupeGuru are built against an older python version in the Windows releases to help with that support (currently 3.7.12 which supports back to Windows 7).  Likely it will be quite some time after Windows 10 is no longer supported before new versions of dupeGuru no longer run on it.

**Reply by Mewto7** [reply]
Hi, hope you have been well. Thank you so much for supporting it. This software is awesome and I am glad I will be able to keep using it.
